docs/structured-streaming-programming-guide.md

```diff
@@ -537,21 +537,21 @@ Most of the common operations on DataFrame/Dataset are supported for streaming.
 <div data-lang="scala"  markdown="1">
 
 {% highlight scala %}
-case class DeviceData(device: String, type: String, signal: Double, time: DateTime)
+case class DeviceData(device: String, deviceType: String, signal: Double, time: DateTime)
```

Comment:
`type` won't compile in scala since it is a keyword, hence here change it to `deviceType`